modules/t/test-genome-DBs/homo_sapiens/core/transcript_attrib.txt

```diff
@@ -1,3 +1,4 @@
 21716	21	ENSG_PARENT_TRANSCRIPT
 21738	24	principal1
 21719	535	NM_030815.3
+21726   576 GENCODE primary
```

Comment:
@sgiorgetti Thanks so much for pointing this out. I ended up editing it on the online editor because my local text editor said they were tabs but they clearly aren't! Test is working now.